packages/core/src/extensions/SuggestionMenu/getDefaultEmojiPickerItems.ts

```diff
@@ -9,13 +9,32 @@ import {
 } from "../../schema/index.js";
 import { DefaultGridSuggestionItem } from "./DefaultGridSuggestionItem.js";
 
-let data:
-  | Promise<{
-      default: EmojiMartData;
-    }>
-  | undefined;
-
+// Temporary fix for https://github.com/missive/emoji-mart/pull/929
```

Comment:
fyi, the reason data was global and could be a promise would be to prevent duplicate loads when `loadEmojiMart` is called twice in a row.

Didn't help readability and it might be a premature optimization so I'm ok with the new version as long as we don't see this causing any issues

Reply:
Ah, I can de-dupe easily. I'll pull that in.